cmd/infra/azure/create.go

```diff
@@ -323,7 +323,7 @@ func (o *CreateInfraOptions) Run(ctx context.Context) (*CreateInfraOutput, error
 	// Extraction is done like this:
 	// docker run --rm -it --entrypoint cat quay.io/openshift-release-dev/ocp-release:4.10.0-rc.0-x86_64 release-manifests/0000_50_installer_coreos-bootimages.yaml |yaml2json |jq .data.stream -r|jq '.architectures.x86_64["rhel-coreos-extensions"]["azure-disk"].url'
 	sourceURL := "https://rhcos.blob.core.windows.net/imagebucket/rhcos-49.84.202110081407-0-azure.x86_64.vhd"
-	blobName := "rhcos.vhd"
+	blobName := "rhcos.x86_64.vhd"
```

Comment:
how about we do the other way around and change `func bootImage` so we image name is not arch specific meaning default image path could work for any arch?

Reply:
But an image is always arch-specific, I'd rather be explicit about that than trying to hide it?